src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -146,6 +148,23 @@ public static Language ParseSubtitleLanguage(string fileName)
 
             return Language.Unknown;
         }
+        
+        public static IEnumerable<string> ParseLanguageTags(string fileName)
+        {
+            try
+            {
+                var simpleFilename = Path.GetFileNameWithoutExtension(fileName);
+                var match = SubtitleLanguageRegex.Match(simpleFilename);
+                var languageTags = from Capture tag in match.Groups["tags"].Captures select tag.Value.ToLower();
+                return languageTags.Where(tag => !tag.Empty());
```

Comment:
Can empty tags be filtered before they're added to the enumerable?

Reply:
Yes - [b3ce4b9](https://github.com/Sonarr/Sonarr/pull/4924/commits/b3ce4b93fb91d0e3b5e8afac1c18b0f846257f64)